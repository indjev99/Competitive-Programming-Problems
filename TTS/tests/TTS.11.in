87 9
6454.65 7637.9
5008.69 -8655.23
9937.31 1118.02
-7025.72 7116.13
9944.56 -1051.55
-700.74 9975.42
-6170.18 7869.49
-9918.41 -1274.8
-9114.16 -4114.87
7089.58 -7052.5
-5077.65 -8614.96
-9695.51 -2448.88
-4426.37 -8967.01
452.227 -9989.77
-304.526 -9995.36
-395.992 9992.16
-9145.24 4045.32
-756.769 9971.32
7473.67 6644.11
1022.96 9947.54
5698.28 -8217.64
9896.5 -1435.02
-6811.94 -7321.03
-2553.48 9668.49
-9965.21 833.423
3267.6 -9451.07
2065.93 9784.27
5897.66 8075.74
-2439.61 9697.85
5305.01 8476.84
-861.461 -9962.83
3011.08 9535.9
-9904.85 1376.2
-4548.18 8905.84
9876.54 -1566.49
-8228.9 5682.01
-7410.89 -6714.06
7251.32 -6886.1
396.651 -9992.13
-3081.17 9513.48
2261.15 -9741.01
-2986.04 9543.77
4308.64 9024.17
5265.77 8501.27
7362.87 -6766.69
-5528.87 -8332.56
9928.68 -1192.15
-686.216 9976.43
9654.11 2607.32
-9678.04 2517.04
9859.73 1669.02
-791.399 9968.64
3164.02 9486.25
-8115.63 -5842.65
8801 -4747.89
-306.938 -9995.29
163.13 -9998.67
-9267.8 3756.04
-102.576 -9999.47
5596.91 -8287.01
-7011.25 7130.38
9941.54 1079.67
9144.34 -4047.34
9883.81 1520
8982.44 -4394.98
-7427.42 6695.78
1068.22 9942.78
9816.26 -1908.14
-9865.04 -1637.35
9490.54 3151.14
5649.22 -8251.44
-7839.36 -6208.41
-2892.15 -9572.64
-8671.73 -4980.06
-5750.78 -8180.99
119.375 9999.29
-2149.59 -9766.23
-2869.55 9579.44
-8766.45 -4811.38
6454.38 -7638.13
9799.31 -1993.38
8254.18 5645.22
8536.22 5208.92
-9759.32 2180.74
-9167.23 -3995.24
-9016.42 4324.83
1898.78 -9818.08
-1299.72 9782.65
-8142.72 6010.33
-3417.86 -5175.58
-6180.91 -2965.28
-1128.29 -9891.78
4521.98 3153.32
9725.4 -801.81
9283.81 8020.09
2122.04 503.97
