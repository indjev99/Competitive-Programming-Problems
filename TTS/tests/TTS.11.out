19557.302864
